### PR TITLE
Merge ID sets support Packs

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/update_id_set.py [hunk 1]
```diff
@@ -1119,6 +1120,11 @@ def add_to_list(self, object_type: IDSetType, obj):
 
         self._id_set_dict.setdefault(object_type, []).append(obj) if obj not in self._id_set_dict[object_type] else None
 
+    def add_to_dict(self, object_type: IDSetType, obj_name, obj_value):
```
Comment (guykeller) [inline demisto_sdk/commands/common/update_id_set.py:1123]: This name currently feels like it doesn't supply enough information about what this function actually does.
Let's see if we can improve iit

File: demisto_sdk/commands/common/update_id_set.py [hunk 2]
```diff
@@ -1119,6 +1120,11 @@ def add_to_list(self, object_type: IDSetType, obj):
...
+        if not IDSetType.has_value(object_type):
+            raise ValueError(f'Invalid IDSetType {object_type}')
```
Comment (guykeller) [inline demisto_sdk/commands/common/update_id_set.py:1125]: Let's make this a more readable and understandable error

**Review by guykeller — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/common/update_id_set.py (hunk 2 above)
Comment (ChanochShayner) [reply demisto_sdk/commands/common/update_id_set.py:1125]: remove this error message, it always needs to be "Packs"

File: demisto_sdk/commands/common/update_id_set.py (hunk 1 above)
Comment (ChanochShayner) [reply demisto_sdk/commands/common/update_id_set.py:1123]: done

**Review by roysagi — APPROVED** [review]
Add changelog please

**Review by guykeller — APPROVED** [review]

**Pull request merged**